LemonUI/Scaleform/Celebration.cs

```diff
@@ -0,0 +1,502 @@
+﻿using System;
+using System.Collections.Generic;
+#if FIVEM
+using CitizenFX.Core.Native;
+#elif RPH
+using Rage.Native;
+#elif (SHVDN2 || SHVDN3)
+using GTA.Native;
+#endif
+
+namespace LemonUI.Scaleform
+{
+    /// <summary>
+    /// Celebration screen that is shown at the end of a mission or job.
+    /// </summary>
+    /// <footer>
+    /// <a href="https://www.google.com/search?q=LemonUI.Scaleform.Celebration">`Celebration` on google.com</a>
+    /// <a href="https://vespura.com/fivem/scaleform/#MP_CELEBRATION">`MP_CELEBRATION` on vespura.com</a>
```

Comment:
I don't recommend linking to the Vespura site as the page has been outdated for a couple of months.

Reply:
I'll remove this link too